tests/Tests/400_ods_dashboard/400_ods_dashboard.robot

```diff
@@ -33,6 +40,39 @@ Verify Content In RHODS Explore Section
     Check Number Of Displayed Cards Is Correct  expected_data=${EXP_DATA_DICT}
     Check Cards Details Are Correct   expected_data=${EXP_DATA_DICT}
 
+Verify Disabled Cards Can Be Removed
+    [Documentation]     Verifies it is possible to remove a disabled card from Enabled page.
+    ...                 It uses RHOSAK as example to test the feature
+    [Tags]    Sanity
+    ...       ODS-1081    ODS-1092
+    Enable RHOSAK
+    Remove RHOSAK From Dashboard
+    Success Message Should Contain   ${RHOSAK_DISPLAYED_APPNAME}
+    Verify Service Is Not Enabled     app_name=${RHOSAK_DISPLAYED_APPNAME}
+    Capture Page Screenshot     after_removal.png
+
+Verify License Of Disabled Cards Can Be Re-validated
+    [Documentation]   Verifies it is possible to re-validate the license of a disabled card
+    ...               from Enabled page. it uses Anaconda CE as example to test the feature.
+    [Tags]    Sanity
+    ...       ODS-1097
+    [Teardown]    Remove Anaconda Commercial Edition Component
```

Comment:
I don't remember discussing about it. I put tthere because I like to see all the TC settings together soon after the test case title. If you want I can move it to the end

Reply:
done